tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -163,7 +163,7 @@ IncludeByKey.SDW_AMP_FEEDBACK {
 				}
 			]
 
-			passthrough-be [
+			passthrough-capture-be [
```

Comment:
@plbossart this is not the wrong direction. The original passthrough-be is applicable for both directions but the passthrough-capture-be limits to 32-bit support only

Reply:
And btw why is this named passthrough in the first place. This should be 'copier-be' or 'generic-be'. The passthrough property depends on how copiers are connected. A copier does not have in itself any passthrough attribute.